notebooks/learning_graph_structured_data_h2.ipynb

```diff
@@ -0,0 +1,835 @@
+{
```

Comment:
<p>NIT: Mean number of vertices</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/806/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Done